letta/client/client.py

```diff
@@ -2271,12 +2263,9 @@ def send_messages(
 
         # format messages
         messages = self.interface.to_list()
-        if include_full_message:
-            letta_messages = messages
-        else:
-            letta_messages = []
-            for m in messages:
-                letta_messages += m.to_letta_message()
+        letta_messages = []
```

Comment:
shouldn't this be removed if we are passing back `LettaResponse` with `LettaMessage`?

Reply:
done